An in-range update of sinon is breaking the build 🚨

## The dependency [sinon](https://github.com/sinonjs/sinon) was updated from `7.3.0` to `7.3.1`.

🚨 [View failing branch](https://github.com/strongloop/loopback-next/compare/master...strongloop:greenkeeper%2Fdefault%2Fsinon-7.3.1).

This version is **covered** by your **current version range** and after updating it in your project **the build failed**.




sinon is a direct dependency of this project, and **it is very likely causing it to break**. If other packages depend on yours, this update is probably also breaking those in turn.



<details>
<summary>Status Details</summary>

- ❌ **continuous-integration/appveyor/branch:** AppVeyor build failed ([Details](https://ci.appveyor.com/project/strongloop/loopback-next/builds/23440646)).
- ❌ **continuous-integration/travis-ci/push:** The Travis CI build failed ([Details](https://travis-ci.org/strongloop/loopback-next/builds/512772826?utm_source=github_status&utm_medium=notification)).
</details>


---

<details>
<summary>Commits</summary>
<p>The new version differs by 6 commits.</p>
<ul>
<li><a href="https://urls.greenkeeper.io/sinonjs/sinon/commit/3812a7dbcea0cb013e42420cd130c24399863084"><code>3812a7d</code></a> <code>Update docs/changelog.md and set new release id in docs/_config.yml</code></li>
<li><a href="https://urls.greenkeeper.io/sinonjs/sinon/commit/93bef5525f61f2278d4d73ddfb150751ad1bfcaa"><code>93bef55</code></a> <code>Add release documentation for v7.3.1</code></li>
<li><a href="https://urls.greenkeeper.io/sinonjs/sinon/commit/e02c1928fdec079c27e841d98c05365b8ed1d9fb"><code>e02c192</code></a> <code>7.3.1</code></li>
<li><a href="https://urls.greenkeeper.io/sinonjs/sinon/commit/8ee1d352c8c44d90dbabdd9ee78d86996625aa19"><code>8ee1d35</code></a> <code>Update CHANGELOG.md and AUTHORS for new release</code></li>
<li><a href="https://urls.greenkeeper.io/sinonjs/sinon/commit/bc53d82ec45e9c1513747e49f9a49af2dda12867"><code>bc53d82</code></a> <code>Fix security issues</code></li>
<li><a href="https://urls.greenkeeper.io/sinonjs/sinon/commit/1a09166870c7a97998ff24d49f03f1b7cc40ee07"><code>1a09166</code></a> <code>Update @sinonjs/samsam to v3.3.1</code></li>
</ul>
<p>See the <a href="https://urls.greenkeeper.io/sinonjs/sinon/compare/059727bf9db20f277c8b056144e31ecbbce18a44...3812a7dbcea0cb013e42420cd130c24399863084">full diff</a></p>
</details>


<details>
<summary>FAQ and help</summary>

There is a collection of [frequently asked questions](https://greenkeeper.io/faq.html). If those don’t help, you can always [ask the humans behind Greenkeeper](https://github.com/greenkeeperio/greenkeeper/issues/new).
</details>

---


Your [Greenkeeper](https://greenkeeper.io) Bot :palm_tree:

Here is the error message:

```
lerna ERR! npm ci exited 1 in '@loopback/cli'
lerna ERR! npm ci stderr:
npm ERR! cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
npm ERR! 
npm ERR! 
npm ERR! Invalid: lock file's sinon@7.2.7 does not satisfy sinon@^7.3.1
npm ERR! 
```

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -34,7 +34,7 @@
     "request": "^2.87.0",
     "request-promise-native": "^1.0.5",
     "rimraf": "^2.6.2",
-    "sinon": "^7.2.3",
+    "sinon": "^7.3.1",
     "yeoman-assert": "^3.1.1",
     "yeoman-environment": "^2.0.6",
     "yeoman-test": "^1.7.0"

--- a/packages/testlab/package.json
+++ b/packages/testlab/package.json
@@ -27,7 +27,7 @@
     "oas-validator": "^3.1.0",
     "shot": "^4.0.7",
     "should": "^13.2.3",
-    "sinon": "^7.2.2",
+    "sinon": "^7.3.1",
     "supertest": "^4.0.2"
   },
   "devDependencies": {

